proto/desmos/posts/v3/msgs.proto

```diff
@@ -354,4 +377,130 @@ message MsgMovePostResponse {
     (gogoproto.customname) = "PostID",
     (gogoproto.moretags) = "yaml:\"post_id\""
   ];
-}
\ No newline at end of file
+}
+
+// MsgRequestPostOwnerTransfer represent a message used to transfer a post ownership to receiver
+//
+// Since: Desmos 6.0.0
+message MsgRequestPostOwnerTransfer {
+  option (cosmos.msg.v1.signer) = "sender";
+
+  // Id of the subspace that holds the post which ownership should be transfered
+  uint64 subspace_id = 1 [
+    (gogoproto.customname) = "SubspaceID",
+    (gogoproto.moretags) = "yaml:\"subspace_id\""
+  ];
+    
+  // Id of the post which will be transferred
+  uint64 post_id = 2[
+    (gogoproto.customname) = "PostID",
+    (gogoproto.moretags) = "yaml:\"post_id\""
+  ];
+
+  // Address of the post ownership receiver
+  string receiver = 3 [
+    (gogoproto.moretags) = "yaml:\"receiver\"",
+    (cosmos_proto.scalar) = "cosmos.AddressString"
+  ];
+    
+  // Address of the sender who is creating a transfer request
+  string sender = 4 [
+    (gogoproto.moretags) = "yaml:\"sender\"",
+    (cosmos_proto.scalar) = "cosmos.AddressString"
+  ];
+}
+
+// MsgRequestPostOwnerTransferResponse defines the Msg/RequestPostOwnerTransfer response type
+//
+// Since: Desmos 6.0.0
+message MsgRequestPostOwnerTransferResponse {}
+
+// MsgCancelPostOwnerTransfer represents a message used to cancel a outgoing post transfer request
+//
+// Since: Desmos 6.0.0
+message MsgCancelPostOwnerTransfer {
+  option (cosmos.msg.v1.signer) = "sender";
+
+  // Id of the subspace that holds the post for which the request should be canceled
+  uint64 subspace_id = 1 [
+    (gogoproto.customname) = "SubspaceID",
+    (gogoproto.moretags) = "yaml:\"subspace_id\""
+  ];
+    
+  // Id of the post for which the request will be cancelled
+  uint64 post_id = 2 [
+    (gogoproto.customname) = "PostID",
+    (gogoproto.moretags) = "yaml:\"post_id\""
+  ];
+
+  // Address of the transfer request sender
+  string sender = 3 [
+    (gogoproto.moretags) = "yaml:\"sender\"",
+    (cosmos_proto.scalar) = "cosmos.AddressString"
+  ];
+}
+
+// MsgCancelPostOwnerTransferResponse defines the Msg/CancelPostOwnerTransfer response type
+//
+// Since: Desmos 6.0.0
+message MsgCancelPostOwnerTransferResponse {}
+
+// MsgAcceptPostOwnerTransfer represents a message used to accept a incoming post transfer request
+//
+// Since: Desmos 6.0.0
+message MsgAcceptPostOwnerTransfer {
```

Comment:
I think it's better to name them `Msg[Accept/Cancel/Refuse]PostOwnerTransferRequest`, similar to the `Msg[Accept/Cancel/Refuse]DTagTransferRequest` we have

Reply:
Updated